server/main.py

```diff
@@ -9,6 +9,9 @@
 from uuid import UUID, uuid4
 
 from fastapi import FastAPI, WebSocket, WebSocketDisconnect
+from pydantic import BaseModel
+
+from server.errors import KappaCloseCodes
```

Comment:
When running the code in the `server` directory, this raises an import error:
```
ModuleNotFoundError: No module named 'server'
```

Reply:
How are you running the server?

Reply:
`uvicorn main:app --reload` in the `server` directory.

Reply:
Oh I see. I've been running it from the root directory so I use `uvicorn server.main:app --reload`. I've fixed it now though and we can add the instructions in the README later, or should I add that in this PR?

Reply:
I think the instructions can be added later. Resolving the conversation.

Reply:
With the current fix running the code as you were previously doesn't work anymore. :cry:

Reply:
> With the current fix running the code as you were previously doesn't work anymore. 😢

Yeah, I used your method of running the server. Best to stick to one so we are consistent and this doesn't flag as a bug :p